astropy/timeseries/tests/test_sampled.py

```diff
@@ -272,7 +272,7 @@ def test_fold_invalid_options():
 def test_pandas():
     pandas = pytest.importorskip("pandas")
 
-    PANDAS_LT_1_5 = not minversion(pandas, '1.4.999')  # When this was written 1.5 was not released.
+    PANDAS_LT_1_5 = minversion(pandas, '1.5')  # When this was written 1.5 was not released.
```

Comment:
Don't take out `not`
```suggestion
    PANDAS_LT_1_5 = not minversion(pandas, '1.5')
```